### PR TITLE
Ported to .NET Core 1.0 RTM

### LINKED ISSUE #38: Update to RTM of .NET Core

### PR DESCRIPTION
**Pull request by rprouse**
Fixes #38

### CODE REVIEW

File: test/dotnet-test-nunit.test/project.json [hunk 1]
```diff
@@ -10,32 +10,23 @@
...
 
   "frameworks": {
-    "netstandard1.5": {
-      "imports": [
-        "dnxcore50",
-        "netstandard1.5",
-        "portable-net45+win8"
-      ],
+    "netcoreapp1.0": {
+      "imports": "dnxcore50",
```
Comment (jasonwilliams200OK) [inline test/dotnet-test-nunit.test/project.json]: `dnxcore50` is not officially depricated moniker. Reference: https://github.com/dotnet/corefx/blob/96fd6fdbae3ea8c0419999d89908dfd28bcb8136/Documentation/architecture/net-platform-standard.md#deprecated-monikers (the VS2015Update3 project template for netcore console and lib haven't been updated yet accordingly, but this doc is genuine).

You may want to try `"imports": ["dotnet5.6"]` or `"imports": ["portable-net45+win8"]` or (if no individaul works; combine) `"imports": ["dotnet5.6", "portable-net45+win8"]`.

Comment (rprouse) [inline test/dotnet-test-nunit.test/project.json]: Thanks, dotnet5.6 worked.

**Comment by rprouse** [comment]
@jasonwilliams200OK any idea why the Travis builds are failing? I can run the commands that are failing locally on Windows and I can't see a difference between this project and the `nunit.portable.agent` project which is working.

**Comment by jasonwilliams200OK** [comment]
@rprouse, finally figured it out.  :smile: 
Original issue is reported at: dotnet/cli#3658.
Added a workaround which is to be removed when that issue is resolved.

Note that macOS build are hanging due to the issues in CI server:
https://www.traviscistatus.com/incidents/rqb906yypnk0
https://twitter.com/traviscistatus

**Comment by rprouse** [comment]
Ick, but thanks :+1: 

I must say, they are not making it easy to keep up.

**Pull request merged**